src/allmydata/test/test_istorageserver.py

```diff
@@ -0,0 +1,282 @@
+"""
+Tests for the ``IStorageServer`` interface.
+
+Note that for performance, in the future we might want the same node to be
+reused across tests, so each test should be careful to generate unique storage
+indexes.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from future.utils import PY2, bchr
+
+if PY2:
+    # fmt: off
+    from future.builtins import filter, map, zip, ascii, chr, hex, input, next, oct, open, pow, round, super, bytes, dict, list, object, range, str, max, min  # noqa: F401
+    # fmt: on
+
+from random import randrange
+
+from twisted.internet.defer import inlineCallbacks
+from twisted.trial.unittest import TestCase
+
+from foolscap.api import Referenceable
+
+from allmydata.interfaces import IStorageServer
+from .test_system import SystemTestMixin
```

Comment:
Looks like it's time to move `SystemTestMixin` to a proper module (ie, not a test module).